spec/integration/provider/oneview_server_profile_template_c7000_spec.rb

```diff
@@ -69,6 +69,15 @@
       expect(provider.get_transformation).to be
     end
 
+    it 'should be able to get a server profile template with available networks', if: api_version >= 300 do
+      resource['data']['queryParameters'] = {
+        'enclosureGroupUri'     => 'EG2',
```

Comment:
One doubt here , is this enclosureGroupUri  and hardwareTypeUri values are correct or need to have the query parameters of new server profile template ?